pkg/cloudprovider/aws/aws.go

```diff
@@ -1294,8 +1295,8 @@ func (s *AWSCloud) describeLoadBalancer(region, name string) (*elb.LoadBalancerD
 
 	response, err := elbClient.DescribeLoadBalancers(request)
 	if err != nil {
-		if awsError := aws.Error(err); awsError != nil {
-			if awsError.Code == "LoadBalancerNotFound" {
+		if awsError := err.(awserr.Error); awsError != nil {
```

Comment:
I think this will panic if err is not awserr.Error.  I think we need something like:
```awsError, ok := err.(awserr.Error); ok  {```

aws.Error was a helper function which did this (including checking pointer vs non-pointer), but it looks like they got rid of that?